gradle/libs.versions.toml

```diff
@@ -93,8 +92,6 @@ sun-activation-version = "1.2.0"
 error-prone = "3.1.0"
 
 [libraries]
-aws-s3 = { group = "software.amazon.awssdk", name = "s3", version.ref = "aws" }
-aws-sts = { group = "software.amazon.awssdk", name = "sts", version.ref = "aws" }
```

Comment:
Does Iceberg aws bundle also include these jars? Users don't need to add them to the classpath manually?

Reply:
aws bundle includes these jars, and user need to add it manually.